browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -618,6 +618,20 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 				],
 				'vertical': 'true'
 			},
+			{ type: 'separator', id: 'home-slidefunctions-break', orientation: 'vertical' },
+			{
+				'type': 'container',
+				'children': [
+					{
+						'id': 'home-assign-layout',
+						'type': 'toolitem',
+						'text': _UNO('.uno:AssignLayout'),
+						'command': '.uno:AssignLayout',
+						'accessibility': { focusBack: true, combination: 'C', de: null }
```

Comment:
we need unique shortcut (this is already assigned to copy on the same tab), you can see them if you press alt or shift+alt and selected tab

Reply:
yes, and now you can press characters pointed by the "home" tab label. It will show shortcuts for that tab then